themes/aframe/layout/index.ejs

```diff
@@ -25,8 +25,9 @@
 
   <div class="home__examples">
     <ul class="home__examples__nav">
-      <% site.data.examples.examples.forEach(function (example, i) { %>
-        <li><a data-example-link="<%= example.slug %>"><%= example.title %></a></li>
+      <% site.data.examples.examples.forEach(function (example, i) {
+        var className = (i === 0) ? 'home__examples--active' : '' %>
```

Comment:
totally making an assumption that the first example is what loads in first.
@ngokevin ?